include/mitsuba/render/mesh.h

```diff
@@ -32,6 +34,12 @@ class MTS_EXPORT_RENDER Mesh : public Shape<Float, Spectrum> {
          Struct *vertex_struct, ScalarSize vertex_count,
          Struct *face_struct,   ScalarSize face_count);
 
+    /// Create a new mesh from a blender mesh
+    Mesh(const std::string &name,
+        size_t loop_tri_count, size_t loop_tri_ptr,
```

Comment:
Use ``unintptr_t`` to pass pointers that have been cast to an unsigned integer. Of course ``size_t`` has the same bit- and signed-ness, but this communicates the intent better.